exporter/zipkin/zipkin.go

```diff
@@ -50,6 +54,36 @@ func NewExporter(reporter reporter.Reporter, localEndpoint *model.Endpoint) *Exp
 	}
 }
 
+type Options struct {
+	// EndpointHostPort is the host:port combination for the Zipkin service
```

Comment:
What is this and what is the difference between this and ReporterURI?

Reply:
Am not well versed with zipkin except for simplifying this exporter so that I could provide a screenshot for a demo, but I believe that:
* EndpointHostPort is the address of a Zipkin's node
* ReporterURI is the URL of the endpoint specifically where spans will be uploaded
Whethere these too can be disjoint, I am ignorant about that, I'll page the big dawgs @rakyll @adriancole to help me answer this.

Reply:
I don't think it is worth to keep both the `EndpointHostPort` and `ReporterURI`, at zipkin we usually use `endpoint_url`. From the Zipkin POV you just need to know the `ReporterURI` and I'd rename it as `EndpointURI`.